src/view/components/rule-engine/rule-condition-group-component.ts

```diff
@@ -17,7 +17,7 @@ import {ServerSideTypeModel} from "../../../api/rule-engine/ServerSideFieldModel
   selector: 'condition-group'
 })
 @View({
-  template: `<div flex layout="column" class="cw-rule-group">
+  template: `<div class="cw-rule-group">
```

Comment:
We don't need ```flex-direction``` in this parent because his childs ```.cw-condition-group-separator``` and ```cw-conditions``` are a block div and a flex div one on top of the other, just like regular blocks. Here is a screenshot with more info: https://cloudup.com/cTEVDwerXv1